tests/test_blackd.py

```diff
@@ -7,195 +8,199 @@
 
 from tests.util import DETERMINISTIC_HEADER, read_data
 
```

Comment:
I think you can use https://docs.pytest.org/en/7.1.x/how-to/skipping.html#skip-all-test-functions-of-a-class-or-module so you don't have to indent the whole thing